xbmc/cores/paplayer/PAPlayer.cpp

```diff
@@ -373,7 +369,10 @@ bool PAPlayer::QueueNextFileEx(const CFileItem &file, bool fadeIn)
   si->m_finishing = false;
   si->m_framesSent = 0;
   si->m_seekNextAtFrame = 0;
-  si->m_seekFrame = -1;
+  if (si->m_fileItem.HasProperty("audiobook_bookmark"))
```

Comment:
I don't really like that the property is called `audiobook_bookmark` even though it just tells PAPlayer an offset to start playing at. This is not audiobook-specific, is it?

Reply:
What about `audio_starttime` for example?